vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -93,6 +93,7 @@
       <RuntimeTypeInfo Condition="'$(UseFabric)' == 'true'">true</RuntimeTypeInfo>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
+      <DisableSpecificWarnings>4267;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
please do not reintroduce SDL violations

Reply:
We do have this .. It is introduced by code in hermes debugger .. I'm looking into it ..

Reply:
We no longer need this exception.